ember.md

```diff
@@ -67,6 +67,27 @@ foo: undefined
 foo: null
 ```
 
+- 1.4 **Multi prop components should always have a new line per property**
```

Comment:
This is long for a subsection title and this gets repeated in the description. Maybe trim this down to 
1.4 *Multi-property Components*
This should also have a link from the TOC.

Reply:
alright, will do.